examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1037 @@
+{
```

Comment:
<p>You can create this oracle using a unitary matrix, which allows us to have the Boolean function act on all possible inputs at once.</p><p> </p><p>implement a constant oracle, balanced oracle, or neither constant or balanced using a...</p><p> </p><p>The I gate is an “identity-gate”, which essentially leaves the qubit as is.</p><p> </p><p>The CNOT gate is a two qubit gate, it is used to entangle two qubits in superposition together. The CNOT gate takes in a control qubit and decides based on that if it needs to flip the spin of the target qubit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>